### PR TITLE
Crate Deploy

### PR DESCRIPTION
**Pull request by droogmic**

### CODE REVIEW

File: .github/workflows/crate-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: "publish osm2lanes"
+        run: |
+          rustup toolchain install stable
```
Comment (dabreegster) [inline .github/workflows/crate-deploy.yml:15]: Do we not need `hecrj/setup-rust-action@v1`? How is `rustup` already around?

Comment (droogmic) [reply .github/workflows/crate-deploy.yml:15]: https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md#rust-tools

File: .github/workflows/crate-deploy.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+          rustup override set stable
+          cd osm2lanes/
+          cargo publish --dry-run
```
[the remark this answers is not in the document]
Comment (droogmic) [reply .github/workflows/crate-deploy.yml:18]: So I think this is fine as a workflow

File: .github/workflows/crate-deploy.yml (hunk 1 above)
Comment (dabreegster) [reply .github/workflows/crate-deploy.yml:15]: Woo, nice!

**Pull request merged**